build.py

```diff
@@ -404,18 +398,29 @@ def update(args: argparse.Namespace, env: dict[str, str]):
         ]
 
     if args.ios:
+        def _get_opencv_toolchain_file(local_args: argparse.Namespace):
+            if local_args.ios_sysroot == "iphoneos":
+                return (
+                    Path("cmake") / "external" / "opencv" / "platforms" / "iOS" / "cmake" /
```

Comment:
use absolute path so it doesn't limit where build.py can be run from.
```suggestion
                    REPO_ROOT / "cmake" / "external" / "opencv" / "platforms" / "iOS" / "cmake" /
```